tls/test/test_record.py

```diff
@@ -10,9 +10,10 @@
 
 import pytest
 
-from tls.record import (
-    ContentType, ProtocolVersion, TLSCiphertext, TLSCompressed, TLSPlaintext
-)
+from tls._common import enums
+
+from tls.record import (ProtocolVersion, TLSCiphertext, TLSCompressed,
+                        TLSPlaintext)
```

Comment:
pfft I am thinking of filing a ticket and making all the `(...)` consistent al across the repo. I should've been more careful :\